app/packs/src/apps/mydb/elements/list/ElementsList.js

```diff
@@ -117,6 +115,9 @@ export default class ElementsList extends React.Component {
     const { totalElements } = this.state;
     Object.keys(state.elements).forEach((key) => {
       totalElements[key] = state.elements[key]?.totalElements;
+      // if (state.elements[key] !== undefined) {
```

Comment:
Das hier sieht komisch aus, sowohl die auskommentierte Variante als auch das was in Zeile 117 steht: Wieso soll state.elements[key] null sein bzw. mit dem ? abgesichert werden, wenn doch in der Schleife da drüber die keys benutzt werden, die vorhanden sind?!?! Verwirrung...

Reply:
Ja, logisch ist das nicht. Ich denke es gab da so einen Fall, wo das nicht so war. Sonst hätte ich an dieser Stelle nicht getestet. Vielleicht war da auch nur etwas im alt -Store kaputt?
Das auskommentierte entferne ich